stable_baselines3/common/env_util.py

```diff
@@ -36,7 +36,7 @@ def is_wrapped(env: Type[gym.Env], wrapper_class: Type[gym.Wrapper]) -> bool:
 
 
 def make_vec_env(
-    env_id: Union[str, Type[gym.Env]],
+    env_id: Union[str, Type[gym.Env], Callable[..., gym.Env]],
```

Comment:
I assume we could also update `make_atari_env` too, no?

Reply:
Sure, just did it!